src/cryptography/hazmat/bindings/_rust/declarative_asn1.pyi

```diff
@@ -11,7 +11,10 @@ def non_root_python_to_rust(cls: type) -> Type: ...
 # annotations like this:
 class Type:
```

Comment:
Does this actually need to be exposed anymore? I don't think we use it in python.

Reply:
we still use it here: https://github.com/pyca/cryptography/blob/cac6426aafd989e4d8d9c4f6b920a332a5280bba/src/cryptography/hazmat/asn1/asn1.py#L66